vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -35,6 +35,34 @@ function DownloadFileWithRetry {
     }
 }
 
+function Download-FileWithAzCopy {
+    param (
+        $URL,
+        $Dest
+    )
+
+    if (!(Test-Path -Path $global:aksTempDir)) {
+        Write-Log "Creating temp dir for tools of building vhd"
+        New-Item -ItemType Directory $global:aksTempDir -Force
+    }
+
+    Write-Log "Downloading azcopy"
+    Invoke-WebRequest -UseBasicParsing "https://aka.ms/downloadazcopy-v10-windows" -OutFile "$global:aksTempDir\azcopy.zip"
+    Expand-Archive -Path "$global:aksTempDir\azcopy.zip" -DestinationPath "$global:aksTempDir\azcopy" -Force
+
+    $env:AZCOPY_AUTO_LOGIN_TYPE="MSI"
+    $env:AZCOPY_MSI_RESOURCE_STRING=$env:WindowsMSIResourceString
+
+    Invoke-Expression -Command "$global:aksTempDir\azcopy\*\azcopy.exe copy $URL $dest"
+
+}
+
+function Cleanup-TemporaryFiles {
+    if (Test-Path -Path $global:aksTempDir) {
+        Remove-Item -Path $global:aksTempDir -Force -Recurse
```

Comment:
could you help to check that  ` C:\Users\packer\.azcopy\` is also deleted automatically?
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=90293303&view=logs&j=da83e503-f0a7-5e39-4c4e-547ddc065d07&t=c25eccdb-c467-51a1-7682-f93363e2409c&l=1285

Reply:
Fixed. Save the plan and log files to c:\akstemp. https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=90379010&view=logs&j=da83e503-f0a7-5e39-4c4e-547ddc065d07&t=c25eccdb-c467-51a1-7682-f93363e2409c&l=1312